src/Examples/Flowsheets/power_generation/rsofc/rsofc_soec_flowsheet.py

```diff
@@ -2378,155 +2378,11 @@ def optimize_model(fs):
-    )
-
-    # Outlet H2O mole fraction from the fuel side should be > 0.20
-    fs.soec_outlet_h2o_constraint = pyo.Constraint(
-        expr=1e3 * fs.spltf1.inlet.mole_frac_comp[0, "H2O"] >= 1e3 * 0.20
-    )
-
-    # Vary split ratio of preheater flue gas split going to ng/o2 preheaters
-    fs.tag_input["preheat_fg_split_to_oxygen"].unfix()  # optimization
-    fs.tag_input["preheat_fg_split_to_oxygen"].setlb(0.30)
-    fs.tag_input["preheat_fg_split_to_oxygen"].setub(0.99)
-
-    # Average current density limitation constraint
-    @fs.Constraint(fs.time)
-    def average_current_density_constraint(b, t):
-        return fs.soec_stack.solid_oxide_cell.average_current_density[0] / 1000 >= -8
-
-    # Unfix cell potential [cell voltage] (optimization dof)
-    # However ensure that it doesn't stray too far above the thermoneutral
-    # point by constraining the outlet fuel/sweep temp from the soec < 1030 K
-    fs.soec_stack.solid_oxide_cell.potential.unfix()
-    fs.soec_stack.solid_oxide_cell.potential.setlb(1.00)
-    fs.soec_stack.solid_oxide_cell.potential.setub(1.40)
-    fs.oxygen_outlet_temperature_constraint = pyo.Constraint(
-        expr=fs.soec_stack.oxygen_outlet.temperature[0] <= 1030.15
-    )
-    fs.fuel_outlet_temperature_constraint = pyo.Constraint(
-        expr=fs.soec_stack.fuel_outlet.temperature[0] <= 1030.15
-    )
-
-    # Ensure that the temperature gradient across the soec is less than 40K
-    # This is set for the outlets and inlets, and air/fuel side
-    fs.soec_outlet_temperature_constraint = pyo.Constraint(
-        expr=(
-            (
-                fs.soec_stack.oxygen_outlet.temperature[0]
-                - fs.soec_stack.fuel_outlet.temperature[0]
-            )
-            ** 2
-            / 100
-            <= 16
-        )
-    )
-    fs.soec_inlet_temperature_constraint = pyo.Constraint(
-        expr=(
-            (
-                fs.soec_stack.oxygen_inlet.temperature[0]
-                - fs.soec_stack.fuel_inlet.temperature[0]
-            )
-            ** 2
-            / 100
-            <= 16
-        )
-    )
-    fs.soec_oxygen_side_DT_constraint = pyo.Constraint(
-        expr=(
-            (
-                fs.soec_stack.oxygen_outlet.temperature[0]
-                - fs.soec_stack.oxygen_inlet.temperature[0]
-            )
-            ** 2
-            / 100
-            <= 16
-        )
-    )
-    fs.soec_fuel_side_DT_constraint = pyo.Constraint(
-        expr=(
-            (
-                fs.soec_stack.fuel_outlet.temperature[0]
-                - fs.soec_stack.fuel_inlet.temperature[0]
-            )
-            ** 2
-            / 100
-            <= 16
-        )
-    )
-
-    fs.obj = pyo.Objective(expr=1e2 * fs.costing.total_variable_OM_cost[0]
-                           / fs.h2_product_rate_mass[0])
+    base_case_optimization(m, solver)
+    fs = m.soec_fs
 
     options = {
-        "max_iter": 150,
+        "max_iter": 500,
```

Comment:
Do we really need 500 iterations here?

Reply:
This isn't the focus of this PR, but I did make a few changes so I could check that the rSOFC model results haven't changed.

I don't recall making that change, but I did paste working solver options from the base case into the loop for running a series of lower load cases, so I'm going to say yes we need it.  I didn't look at the robustness of the rSOFC case, and I'm not that familiar with it.  I'll com back in later PR and hopfully I can improve it.

The good news is that the results are still the same as what we used in the report.  The bad news is we need to take another look at the robustness of the model.